pkg/crd/operator/v1alpha1/configuration.go

```diff
@@ -312,6 +313,9 @@ type KubermaticVersioningConfiguration struct {
 	// updates as well. 'automaticNodeUpdate: true' implies 'automatic: true' as well,
 	// because Nodes may not have a newer version than the controlplane.
 	Updates []Update `json:"updates,omitempty"`
+
+	// ProviderIncompatibilities lists all the kubernetes version prohibited for a given provider. Wildcards are allowed, e.g. "1.22.*".
```

Comment:
> Wildcards are allowed, e.g. "1.22.*".

That is a statement that belongs on the "Version" field in the `Incompatibiliy` struct.

Reply:
right, gonna change it